rage/src/bin/rage-mount/cli.rs

```diff
@@ -51,5 +53,6 @@ pub(crate) struct AgeMountOptions {
     #[arg(short, long)]
     #[arg(value_name = fl!("identity"))]
     #[arg(help = fl!("help-flag-identity"))]
+    #[arg(value_hint = ValueHint::FilePath)]
```

Comment:
Does this correctly auto-complete each instance of `-i/--identity` as a filepath (i.e. if `-i/--identity` is repeated, it starts autocompletion again)?

Reply:
It does (at least on zsh).